lib/std/Build/Step/Run.zig

```diff
@@ -1001,6 +1001,25 @@ fn spawnChildAndCollect(
         child.stdin_behavior = .Pipe;
     }
 
+    switch (self.stdin) {
+        .lazy_path => |lazy_path| {
+            const path = lazy_path.getPath(self.step.owner);
+            const file = self.step.owner.build_root.handle.openFile(path, .{}) catch |err| {
+                return self.step.fail("unable to open stdin file: {s}", .{@errorName(err)});
+            };
+            // defer file.close();
+
+            // child.stdin.?.writeFileAll(file, .{}) catch |err| {
+            //     return self.step.fail("unable to write file to stdin: {s}", .{@errorName(err)});
+            // };
+            // child.stdin.?.close();
+            // child.stdin = null;
```

Comment:
let's not leave commented code that clutters up readability?

Reply:
Whoopsies, got it